CONTRIBUTING.md

```diff
@@ -1,16 +1,30 @@
 # mongodb-operator
 
-## Code overview
 
-TODO: expand on this section when spec doc is created
+## Developing
 
-## Intended use case
+As this is a machine charm, it can be deployed locally using [Juju on a local LXD cloud](https://juju.is/docs/olm/lxd). Once the controller is created, to deploy this workload you need to follow some preliminar steps.
 
```

Comment:
If you do not already have a Juju controller bootstrapped to an LXD `localhost` cloud you can set one up by doing the following:

```bash
# install requirements 
sudo snap install charmcraft --classic
sudo snap install lxd
sudo snap install juju --classic

# configure lxd
sudo adduser $USER lxd
newgrp lxd
lxd init --auto
lxc network set lxdbr0 ipv6.address none

# boostrap controller to lxd
juju clouds
juju bootstrap localhost overlord
```